src/constants.js

```diff
@@ -24,12 +24,18 @@ export const SAMPLED_MASK = 0x1;
 // DEBUG_MASK is the bit mask indicationg that a span has been marked for debug.
 export const DEBUG_MASK = 0x2;
 
+// DEFERRED_SAMPLING_MASK is the bit mask indicating that a span has been marked for deferred sampling.
```

Comment:
I'll update the first sentence and remove the example about mobile. 
However, I haven't claimed that such a feedback loop cannot exist.